src/intervals/arithmetic.jl

```diff
@@ -165,25 +184,29 @@ function fma{T}(a::Interval{T}, b::Interval{T}, c::Interval{T})
     if isentire(a)
         b == zero(b) && return c
         return entireinterval(T)
+
     elseif isentire(b)
         a == zero(a) && return c
         return entireinterval(T)
+
     end
 
     lo = setrounding(T, RoundDown) do
         lo1 = fma(a.lo, b.lo, c.lo)
         lo2 = fma(a.lo, b.hi, c.lo)
         lo3 = fma(a.hi, b.lo, c.lo)
         lo4 = fma(a.hi, b.hi, c.lo)
-        min(lo1, lo2, lo3, lo4)
+        min_ignore_nans(lo1, lo2, lo3, lo4)
```

Comment:
By the way, maybe it is wrong that `Interval` allows to do `Interval(NaN, 2)` at all?